Assets/Scripts/Entities/Player/WeaponHandler.cs

```diff
@@ -13,16 +13,21 @@ public class WeaponHandler : MonoBehaviour
         private GameObject _currentWeapon;
         private GameObject _selectedWeapon;
         private AnimationHandler _animHandler;
+        private bool isPlayer;
 
         public void Start()
         {
             _animHandler = GetComponent<AnimationHandler>();
+            isPlayer = GetComponent<Entity>().tag == "Player";
```

Comment:
kan veranderen naar gameObject.tag == "Player"
component oproepen is niet nodig